### PR TITLE
[ROOT 6-22] Get 6-22 branch for root next

### PR DESCRIPTION
**Pull request by mrodozov**
please test

### CODE REVIEW

**Comment by smuzaffar** [comment]
@mrodozov , I think its better to use root.spec from rootmaster here. There we have enabled pyroot and build for both python versions

**Comment by smuzaffar** [comment]
also check what other changes we have in rootmaster branch and if we have to get them back in here e.g. yoda might need updates too

File: root.spec [hunk 1]
```diff
@@ -39,7 +40,7 @@ export CXXFLAGS=-D__ROOFIT_NOBANNER
 
 cmake ../%{n}-%{realversion} \
   -G Ninja \
-  -DCMAKE_BUILD_TYPE=Release \
+  -DCMAKE_BUILD_TYPE=Debug \
```
Comment (smuzaffar) [inline root.spec]: @mrodozov , I would suggest to build it in Release mode

Comment (mrodozov) [reply root.spec]: i was going to ask if the new cmake was fixing this 😅 it's also need to be changed in master

Comment (smuzaffar) [reply root.spec]: No, for master we need to build in Debug mode.

**Comment by mrodozov** [comment]
please test

File: root.spec (hunk 1 above)
Comment (smuzaffar) [reply root.spec]: issue with cmake was that it was not allowing us to build with `-DCMAKE_BUILD_TYPE=Debug -DLLVM_BUILD_TYPE=Release` (i.e. one with Release and other with Debug). For root master we want to build with `-DCMAKE_BUILD_TYPE=Debug`  bit for normal IBs and rootnext we want to be in `Release` mode (i.e. close to the normal IBs)

**Comment by smuzaffar** [comment]
test parameters
- full_cmssw = true

**Comment by smuzaffar** [comment]
@mrodozov , you should test this in full release mode :-)

**Comment by mrodozov** [comment]
please test

**Comment by mrodozov** [comment]
abort

**Comment by mrodozov** [comment]
please test

**Comment by smuzaffar** [comment]
+externals

**Pull request merged**